pyomo/contrib/doe/tests/test_example.py

```diff
@@ -44,21 +44,24 @@
 class TestReactorExample(unittest.TestCase):
     @unittest.skipIf(not ipopt_available, "The 'ipopt' command is not available")
     @unittest.skipIf(not scipy_available, "scipy is not available")
+    @unittest.skipIf(not numpy_available, "Numpy is not available")
     def test_reactor_compute_FIM(self):
-        from pyomo.contrib.doe.example import reactor_compute_FIM
+        from pyomo.contrib.doe.examples import reactor_compute_FIM
 
         reactor_compute_FIM.main()
 
     @unittest.skipIf(not ipopt_available, "The 'ipopt' command is not available")
+    @unittest.skipIf(not numpy_available, "Numpy is not available")
     def test_reactor_optimize_doe(self):
-        from pyomo.contrib.doe.example import reactor_optimize_doe
+        from pyomo.contrib.doe.examples import reactor_optimize_doe
 
         reactor_optimize_doe.main()
 
     @unittest.skipIf(not ipopt_available, "The 'ipopt' command is not available")
     @unittest.skipIf(not pandas_available, "pandas is not available")
+    @unittest.skipIf(not numpy_available, "Numpy is not available")
     def test_reactor_grid_search(self):
-        from pyomo.contrib.doe.example import reactor_grid_search
+        from pyomo.contrib.doe.examples import reactor_grid_search
```

Comment:
These tests aren't asserting anything, they just confirm that the examples run without errors. Please add a TODO item to the Pyomo.DoE development issue to strengthen these tests by checking if the results are correct.

Reply:
I didn't add assertions for them now since test_reactor_example.py has assertions to see results. I add it to #2610